docs/new_deployment.md

```diff
@@ -47,3 +56,25 @@ If everything works, try to access the website from the configured domain.
 ## Restrict Access
 
 Follow the [IAP setup](./iap.md) to use Cloud IAP to restrict access to the new instance.
+
+## Adding Data
+Each data import should be contained within a top-level folder in the GCS bucket associated with the instance. Within each folder, there should be exactly one `manifest.json`, exactly one TMCF file, and one or more CSV files. 
+
+Each `manifest.json` should be structured as follows: 
+
+```
+{
+    "importName": "<IMPORT_NAME>",
+    "provenanceUrl": "<PROVENANCE_URL>",
+    "dataDownloadUrl": "<DATA_DOWNLOAD_URL>"
+}
+```
+
+`<IMPORT_NAME>` will be used to group the Statistical Variables from the import in the website tree hierarchy navigation.
+
+Each TMCF file should have one node entry for each Statistical Variable in the import.
+
+CSV files should be structured so that each Statistical Variable has its own column. The first few columns should be reserved for other StatVarObservation properties, such as  `observationAbout`, `observationDate`, `observationPeriod`, `unit`, etc. 
+
+## Merging Configs
+When submitting configs for a new instance, please include `/deploy/gke/<new_instance>.yaml`, `deploy/overlays/<new_instance>/kustomization.yaml` (delete the images before submitting), and `deploy/overlays/<new_instance>/patch.yaml`, if needed. Do not include changes to `mixer_hash.txt`, `website_hash.txt`, or the generated `deploy/overlays/<new_instance>/<new_instance>.yaml`.
```

Comment:
Can remove the text starting from "(delete the images before submitting)..."

Reply:
done